digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -1894,7 +1850,7 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
           logger.info(s"Current status is $currentState, fetching TR state")
           val trProjectState = ViiteTierekisteriClient.getProjectStatusObject(trId)
```

Comment:
Tämä on turha kutsu.

Reply:
Poisto aiheuttaisi cascadepoistoja. Teenkö silti tässä, vai sopiiko jättää VIITE-2622:n puolelle?

Reply:
siis jos vaan poistaa tämän kutsun tästä kohtaa prosessia?

Reply:
trProjectState ei enään tarvita mihinkään, niin sitä on turha hakeakaan.

Reply:
Jos poistaa vain tämän, ei käänny. :)
Myös vanhat rivit 1898 ja sen myötä 1899, sekä 1900 funktiokutsuineen päätyvät tällöin siivoukseen.

Reply:
Jaa no, voin tietty antaa virheviestiksi "". Ok!